### PR TITLE
Add gateway location tests

### PR DESCRIPTION
**Pull request by asmulko**
#### Summary

References #3606 

#### Changes

- Added gateway location e2e tests

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: cypress/integration/console/gateways/location/create.spec.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      .should('be.visible')
+      .findByText(`Location updated`)
+      .should('be.visible')
+    cy.findByTestId('error-notification').should('not.exist')
+    cy.findByTestId('full-error-view').should('not.exist')
+    cy.location('pathname').should(
+      'eq',
+      `${Cypress.config('consoleRootPath')}/gateways/${gatewayId}/location`,
+    )
+  })
```
Comment (kschiffer) [inline cypress/integration/console/gateways/location/create.spec.js]: There's no reason to assume that the location will have changed as a result of the submit, so these checks can be removed.

File: cypress/integration/console/gateways/location/edit.spec.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    cy.findByTestId('error-notification').should('not.exist')
+    cy.findByTestId('toast-notification')
+      .should('be.visible')
+      .findByText(`Location deleted`)
+      .should('be.visible')
+    cy.location('pathname').should(
+      'eq',
+      `${Cypress.config('consoleRootPath')}/gateways/${gatewayId}/location`,
+    )
```
Comment (kschiffer) [inline cypress/integration/console/gateways/location/edit.spec.js]:
```suggestion
```

**Review by kschiffer — CHANGES_REQUESTED** [review]
Can you add a test that checks whether the location can be set by clicking into the map? This should be possible by selecting the map widget and using the `.click()` method. Note that you can [use this method with coordinates](https://docs.cypress.io/api/commands/click.html#Syntax).

**Comment by asmulko** [comment]
> Can you add a test that checks whether the location can be set by clicking into the map? This should be possible by selecting the map widget and using the `.click()` method. Note that you can [use this method with coordinates](https://docs.cypress.io/api/commands/click.html#Syntax).

Thanks for the information on how to do it, I've added the test

**Review by kschiffer — APPROVED** [review]

**Pull request merged**